packages/next/src/components/lightbox-overlay.tsx

```diff
@@ -0,0 +1,86 @@
+import type { ReactNode } from 'react';
+
+/**
+ * Renders the lightbox overlay for displaying an enlarged image with interactions.
+ *
+ * @return The lightbox overlay.
+ */
+export function LightboxOverlay(): ReactNode {
+	const closeButtonLabel = 'Close';
+	const closeButtonColor = 'var(--wp--preset--color--contrast)';
+	const backgroundColor = 'var(--wp--preset--color--base)';
```

Comment:
Add a comment explaining what these are, why theyre hardcoded, and where they're coming from.